packages/odata-common/README.md

```diff
@@ -3,7 +3,7 @@
 <a href="https://sap.com/s4sdk"><img src="https://help.sap.com/doc/2324e9c3b28748a4ae2ad08166d77675/1.0/en-US/logo-with-js.svg" alt="SAP Cloud SDK for JavaScript Logo" height="122.92" width="226.773"/></a>
 <!-- sap-cloud-sdk-logo-stop -->
 
-# @sap-cloud-sdk/odata-common/internal
+# @sap-cloud-sdk/odata-common
```

Comment:
👍 
Should we add a note to the users, saying `odata-vX` should be used as a complete library instead of this internal package?

Reply:
Makes sense.